lib/mix/tasks/phoenix.build_templates.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Mix.Tasks.Phoenix.BuildTemplates do
```

Comment:
It is an extra file I accidentally copied from phoenix. Give me a second, and I'll remove it.